lib/common/database.dart

```diff
@@ -676,39 +888,39 @@ class DatabaseService {
         .get();
   }
 
-  /// Search function to query task tags
-  /// Takes a query string and value to limit number of outputs
-  /// Returns list of tasks that the query is in the tags of (not substring), with amount specified by limit
-  Future<List<Task>> searchTaskTags(String query, int limit) async {
-    // tags are a little weird since they are stored as a list, and firestore doesn't support substring searching for complex types
-    // seems like the best they have available without 3rd party solutions is array-contains
-    final QuerySnapshot<Map<String, dynamic>> queriedTaskResults =
-        await _tagQuery(query, limit, "tasks");
-    return _snapshotToTasks(queriedTaskResults);
-  }
-
-  /// Search function to query event tags
-  /// Takes a query string and value to limit number of outputs
-  /// Returns list of events that the query is in the tags of (not substring), with amount specified by limit
-  Future<List<Event>> searchEventTags(String query, int limit) async {
-    final QuerySnapshot<Map<String, dynamic>> queriedEventResults =
-        await _tagQuery(query, limit, "events");
-    return _snapshotToEvents(queriedEventResults);
-  }
```

Comment:
Is there no longer a way to search for them then?

Reply:
it's `getUndertakingsWithTag`. I can add some wrappers, but it works for now.